urllib3/__init__.py

```diff
@@ -89,6 +93,8 @@ def add_stderr_logger(level=logging.DEBUG):
 
 
 def disable_warnings(category=exceptions.HTTPWarning):
+    # type: (type) -> None
+    # TODO be more precise about which type is accepted
```

Comment:
I guess it will require either a typeshed or PEP change, `type` is working for now.